documentation/governance.md

```diff
@@ -140,9 +140,9 @@ Anyone who is a current or potential publisher or user of the 360Giving Standard
 ```eval_rst
 **Footnotes**
 
-.. [1]  For a list of current Stewardship Committee members and their Terms of Reference, visit: http://www.threesixtygiving.org/governance-of-the-360giving-standard/.
+.. [1]  For a list of current Stewardship Committee members and their Terms of Reference, visit: https://www.threesixtygiving.org/governance-of-the-360giving-standard/.
 .. [2] See https://open-stand.org/about-us/principles/.
 .. [3] See https://en.wikipedia.org/wiki/Deprecation
-.. [4] See http://cove.opendataservices.coop/360/.
+.. [4] See https://cove.opendataservices.coop/360/.
```

Comment:
I think this should just be a link to https://dataquality.threesixtygiving.org/ , for two reasons:
- This link doesn't work any more
- The DQT link is where we tell people to do in any other case

Thoughts @mrshll1001 @drkane ?

It's still non-normative, because the intention here was clearly to just link to the DQT.

Reply:
I think it should be that as well. there's no change to the meaning of the link so we can do this

Reply:
Done as per 5a03ac1